server/tests/runs.e2e-spec.ts

```diff
@@ -0,0 +1,376 @@
+                    createMany: {
+                        data: [{ zoneNum: 1 }, { zoneNum: 2 }]
+                    }
+                },
+                createdAt: new Date('2018-10-05 11:37:00')
+            }
+        });
+
+        run3 = await prisma.run.create({
+            data: {
+                map: { connect: { id: map2.id } },
+                player: { connect: { id: user1.id } },
+                trackNum: 1,
+                zoneNum: 1,
+                ticks: 123456,
+                tickRate: 66.6,
+                flags: 1 << 4,
+                file: 'surfing_fast.run',
+                hash: '0x1137',
+                baseStats: {
+                    create: {
+                        jumps: 1
+                    }
+                },
+                zoneStats: {
+                    createMany: {
+                        data: [{ zoneNum: 1 }, { zoneNum: 2 }]
+                    }
+                }
+            }
+        });
+
+        run4 = await prisma.run.create({
+            data: {
+                map: { connect: { id: map2.id } },
+                player: { connect: { id: user2.id } },
+                trackNum: 1,
+                zoneNum: 1,
+                ticks: 99999,
+                tickRate: 66.6,
+                flags: 1 << 5,
+                file: 'surfing_faster.run',
+                hash: '0xh4sh',
+                rank: {
+                    create: {
+                        gameType: MapType.RJ,
+                        flags: 0,
+                        trackNum: 0,
+                        zoneNum: 0,
+                        rank: 111,
+                        map: { connect: { id: map2.id } },
+                        user: { connect: { id: user2.id } }
+                    }
+                },
+                baseStats: {
+                    create: {
+                        jumps: 1
+                    }
+                },
+                zoneStats: {
+                    createMany: {
+                        data: [{ zoneNum: 1 }, { zoneNum: 2 }]
+                    }
+                }
+            },
+            include: {
+                baseStats: true,
+                zoneStats: true
+            }
+        });
+
+        const authService: AuthService = global.auth as AuthService;
+        global.accessToken = (await authService.login(user1)).access_token;
+    });
+
+    afterEach(async () => {
+        const prisma: PrismaService = global.prisma;
+
+        await prisma.user.deleteMany({ where: { id: { in: [user1.id, user2.id] } } });
+        await prisma.run.deleteMany({ where: { id: { in: [run1.id, run2.id, run3.id, run4.id] } } });
+        await prisma.map.deleteMany({ where: { id: { in: [map1.id, map2.id] } } });
+    });
+
+    describe('GET /api/v1/runs', () => {
+        const expects = (res) => expect(res.body).toBeValidPagedDto(RunDto);
+        const filter = (x) => {[run1.id, run2.id, run3.id, run4.id].includes(x.id)};
```

Comment:
What's the filter being used for? Looks like it should be being used in the `expandTest`s